app/controllers/presence_controller.rb

```diff
@@ -42,8 +42,6 @@ def get
   end
 
   def update
-    raise Discourse::ReadOnly if @readonly_mode
```

Comment:
This is not needed anymore since this action is a `POST` and is already handled by the `read_only_mixin`.